cmd/scollector/main.go

```diff
@@ -39,6 +39,7 @@ var (
 	flagBatchSize       = flag.Int("b", 0, "OpenTSDB batch size. Used for debugging bad data.")
 	flagSNMP            = flag.String("s", "", "SNMP host to poll of the format: \"community@host[,community@host...]\".")
 	flagICMP            = flag.String("i", "", "ICMP host to ping of the format: \"host[,host...]\".")
+	flagHAProxy         = flag.String("haproxy", "", `haproxy credentials, of the format: "user:password"`)
```

Comment:
if `haproxy_instance` can only be supplied via config file, why do we need a flag for the credentials? Is it just to keep sensitive info out of the config file? I would rather not add flags if there is no way to use this feature without a config file anyway.